### ISSUE TITLE
Giving top down entity a base class causes problems

### ISSUE THREAD

**Issue by vchelaru** [vchelaru/FlatRedBall#1399]
Giving the Creature (formerly Player) entity a base class called CreatureBase broke top down movement. I had to manually delete files in Creature and manually edit stuff to remove the animations specific to the child class because they were colliding. I couldn't do this in Glue UI.

Solving the problem described here should only happen after solving issue 1398:
https://github.com/vchelaru/FlatRedBall/issues/1398

**Comment by vchelaru** [comment]
As I've worked on this, I ran into a few bugs and inconveniences so I'm listing them here to be fixed:

- [x] Marking an entity as a platformer creates a new CSV and selects the new CSV. It should not be selected because users probably don't care about this file (at least not at first), and this surprise selection interrupts the workflow.
- [x] Marking an entity as not a platformer or top down should remove the CSV files from the entity. The file can be left on disk for the sake of not wiping user content, but remove it from Glue so that inheritance can be properly set up.
- [x] Marking an entity as a platformer, then not as a platformer should remove the platformer variables.
